pisound-btn/pisound-btn.c

```diff
@@ -953,10 +993,11 @@ static void print_usage(void)
 		"\t--help               Display the usage information.\n"
 		"\t--version            Show the version information.\n"
 		"\t--gpio               The pin GPIO number to use for the button. Default is 17.\n"
+		"\t--active_low         Reverse the sense of the active state.  Normally active is when GPIO goes high\n"
 		"\t--conf               Specify the path to configuration file to use. Default is /etc/pisound.conf.\n"
 		"\t--press-count-limit  Set the press count limit. Use 0 for no limit. Default is 8.\n"
 		"\t--debug <n>          Enable debugging at level n (higher value = more logging)"
-		"\t-n                   Short for --click-count-limit.\n"
+		"\t-n                   Short for --press-count-limit.\n"
```

Comment:
Actually looks like --click-count-limit is currently the option that's handled in argv handling code, so the --press-count-limit should be changed to --click-count-limit instead. (or argv handling fixed too. I don't think there's any other users of this option yet, so up to you what is the correct argument name :) )